docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -96,14 +96,51 @@ releases of Ansible, there can sometimes be exceptions for critical issues.
 
 .. _GitHub: https://github.com/ansible/ansible
 
+.. _changelogs_how_to:
+
 Changelogs
 ~~~~~~~~~~~~~~~~~~
 
-Since 2.5, we've logged changes to ``stable-<version>`` git branches at ``stable-<version>/changelogs/CHANGELOG-v<version>.rst``.
-For example, here's the changelog for 2.5_ on GitHub.
```

Comment:
Please leave these two lines in the docs - they're helpful for anyone who hits this page because they're searching for something like the 2.6 changelog.